src/util/worker/mod.rs

```diff
@@ -193,9 +261,14 @@ impl<T: Display + Send + 'static> Worker<T> {
             scheduler: Scheduler::new(name, AtomicUsize::new(0), tx),
             receiver: Mutex::new(Some(rx)),
             handle: None,
+            tasks_per_tick: None,
         }
     }
 
+    pub fn enable_tick_task(&mut self, tasks_per_tick: usize) {
```

Comment:
How about using this after `start` function? I don't think this is a good way to construct the worker.